odoo/tools/translate.py

```diff
@@ -530,6 +530,8 @@ def next(self):
                     line = line[2:].strip()
                     if not line.startswith('module:'):
                         comments.append(line)
+                    else:
+                        module = line.split()[1]
```

Comment:
This assumes that there is a whitespace after the colon.  Wouldn't this be safer?
```suggestion
                        module = line[7:].strip()
```